types/cjs.d.ts

```diff
@@ -31,6 +31,8 @@ declare namespace megajs {
     inbox: MutableFile
     mounts: MutableFile[]
     files: { [id in string]: MutableFile }
+    filter (query: string | string[] | ((file: MutableFile) => void), deep?: boolean): MutableFile | undefined
```

Comment:
I don't know a lot about TypeScript, but shouldn't those filter/query functions return booleans?

Reply:
Yes you are correct. The query function should return a boolean.
Fixed.